src/gui/sharedialog.cpp

```diff
@@ -275,6 +275,13 @@ void ShareDialog::getShares()
     job->addPassStatusCode(404); // don't report error if share doesn't exist yet
     connect(job, SIGNAL(jobFinished(QVariantMap)), this, SLOT(slotSharesFetched(QVariantMap)));
     job->start();
+
+    if (QFileInfo(_localPath).isFile()) {
+        QUrl url2 = Account::concatUrlPath(_account->url(), QString("index.php/apps/files/api/v1/thumbnail/150/150/%1").arg(_sharePath));
```

Comment:
using `Qlatin1String("index.php/apps/files/api/v1/thumbnail/150/150/") + _sharedPath`
Is both more readable (at least for me) and efficient that  `QString("...").arg(...)`